src/test/resources/function-tests.json

```diff
@@ -624,8 +624,8 @@
    },
    {
       "input" : "{}",
-      "output" : "101574",
-      "query" : "hash-code(\"foo\")"
+      "output" : "\"2c26b46b68ffc68ff99b453c1d30413413422d706483bfa0f98a5e886266e7ae\"",
+      "query" : "sha256(\"foo\")"
    },
```

Comment:
It would be good to also have tests for the `null` case, as well as some integer, and the same integer as a string. If we pass an integer to the function it should be converted to a string and then hashed. The two cases with integers can also be added as examples.